app/controllers/documents_controller.rb

```diff
@@ -1,58 +1,57 @@
 # encoding:UTF-8
 class DocumentsController < ApplicationController
-  
-  before_filter :authenticate, only: [:update,:edit,:new,:destroy,:create]
-  before_action :set_document, only: [:show,:update,:edit,:destroy]
-  before_action :set_edit  
-  
+
+  before_filter :authenticate, only: [:update, :edit, :new, :destroy, :create]
+  before_action :set_document, only: [:show, :update, :edit, :destroy]
+  before_action :set_edit
+
   def index
-    if(current_user)
+    if (current_user)
       @logged_in = true
       @documents = Document.all
     else
-      @documents = Document.publik      
+      @documents = Document.publik
     end
     @documents_grid = initialize_grid(@documents)
   end
-  
+
   def new
     @dokument = Document.new
   end
-  
+
   def show
-    if(!@dokument.public) && (!current_user)
+    if (!@dokument.public) && (!current_user)
       redirect_to action: :index
       @documents = Document.publik
       @documents_grid = initialize_grid(@documents)
       return
-    elsif((!@dokument.public) && (current_user) && (@dokument.pdf_file_name) || (@dokument.public))
-      send_file(@dokument.pdf.path, filename:@dokument.pdf_file_name, type: "application/pdf",disposition: 'inline',x_sendfile: true)
+    elsif ((!@dokument.public) && (current_user) && (@dokument.pdf_file_name) || (@dokument.public))
+      send_file(@dokument.pdf.path, filename: @dokument.pdf_file_name, type: "application/pdf", disposition: 'inline', x_sendfile: true)
       return
     end
   end
-  def edit    
+
+  def edit
   end
-  
+
   def create
     @user = current_user
-    if(@user)
-      if(@user.profile)
-        @dokument = Document.new(document_params) 
-        @dokument.update(profile_id: @user.profile.id)
-        @dokument.pdf = params[:document][:pdf]       
-        respond_to do |format|
-          if @dokument.save
-            format.html { redirect_to documents_path, :notice => 'Dokumentet skapades!' }            
-          else
-            format.html { render action: "new" }            
-          end        
+    if (@user)
+      @dokument = Document.new(document_params)
+      @dokument.update(user_id: @user.id)
+      @dokument.pdf = params[:document][:pdf]
+      respond_to do |format|
+        if @dokument.save
+          format.html { redirect_to documents_path, :notice => 'Dokumentet skapades!' }
+        else
+          format.html { render action: "new" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.